apps/src/templates/CodeWorkspace.jsx

```diff
@@ -1,3 +1,4 @@
+import $ from 'jquery';
```

Comment:
Any idea how close we are to removing [this global `$` rule](https://github.com/code-dot-org/code-dot-org/blob/staging/apps/.eslintrc.js#L16)?